### ISSUE TITLE
`gup` issues (dev workflow)

### ISSUE THREAD

**Issue by eelstork** [active-logic/howl#53]
`gup` should be automating grammar updates, but this does not work as well as it should.

```
+ cd /Users/xxx/.atom/packages/language-howl/
+ apm install --verbose
Installing modules audited 2 packages in 0.446s
found 0 vulnerabilities
✓
+ git add --all
+ git commit -m patch
On branch master
Your branch is ahead of 'origin/master' by 2 commits.
  (use "git push" to publish your local commits)
nothing to commit, working tree clean
```

**Comment by eelstork** [comment]
Workflow:

```
cd          ~/.atom/packages/language-howl/
apm         install --verbose
git         add --all
git         commit -m "$1"
git         push
apm         publish $1
```

Indeed there *should not* be anything to commit here, provided the atom package refers `latest` for tree-sitter package version, and derived products are not checked in.
In any case we probably do not want to publish the apm package at this point, because the grammar and style must be updated first.

**Comment by eelstork** [comment]
Fixed `gup` and documented workflow in 612a778
